eng/Versions.props

```diff
@@ -233,12 +237,15 @@
     <MicrosoftWebAdministrationPackageVersion>11.1.0</MicrosoftWebAdministrationPackageVersion>
     <MicrosoftWebXdtPackageVersion>1.4.0</MicrosoftWebXdtPackageVersion>
     <SystemIdentityModelTokensJwtPackageVersion>6.7.1</SystemIdentityModelTokensJwtPackageVersion>
-    <!-- Packages from 2.1/2.2 branches used for site extension build -->
+    <!-- Packages from 2.1, 2.2, 3.1 and 5.0 branches used for site extension build. -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
```

Comment:
Should this be 2.2.5 now? Or, can this package be removed here, in Dependencies.props and in Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj❔

Reply:
I'm all for removing anything 2.2
Lets ping some folks to make sure they're ok with us removing 2.2 from the site extension
@Pilchie @shirhatti @bradygaster